index.html

```diff
@@ -4,9 +4,26 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <title>Oshane Design Portfolio</title>
 </head>
-<body>
-    <h1>Text</h1>
+<body> 
+    <div class="main-Container"> 
+    <header class="navBar">
+        <p class="my-logo">My Logo</p>
+        <img class="menuDropDown" src="images/Union.svg" alt="menuDropDown">
+    </header>
```

Comment:
- [ ] Please, the logo should be an `a` tag since it should take you to the home page when you click on it, so you can change this p tag to look something like this ` <a href="#">My Logo</a>` for now, and please wrap the hamburger `img` tag in a `nav` tag instead.
- [ ] Please always play with the properties given in the figma design, your logo should look like this, see the image below ⬇️ 

<img width="286" alt="fig" src="https://user-images.githubusercontent.com/79058364/153514348-8f9145f8-c4b1-4bb1-8c08-fef395f348fa.png">

but instead, it looks like this ⬇️ 

<img width="339" alt="yu" src="https://user-images.githubusercontent.com/79058364/153514439-38683c3d-489b-4c2e-878b-e1a3f210172e.png">

the font family does not match the design at all, please fix that too

Reply:
@BenMukebo resolved.